configure

```diff
@@ -198,6 +169,14 @@ while [ $# -ne 0 ]; do
             echo "${usage}" 1>&2
             exit 1
             ;;
+        --show-config)
+            if [ ! -f "$builddir/CMakeCache.txt" ]; then
+                echo "Error: no CMake build found under '$builddir'." 1>&2
+                exit 1
+            fi
+            grep -E "^ENABLE_|^ZEEK_|^INSTALL_|^CMAKE_INSTALL_PRE|^CMAKE_C.*_FLAGS|^CMAKE_BUILD" "$builddir/CMakeCache.txt" | grep -v ':INTERNAL'
```

Comment:
I wonder whether it would be useful to include `CMAKE_CXX_COMPILER`/`CMAKE_C_COMPILER` here. Right now this is injected through `CXX=... ./configure` and it not showing up in `build/config.status` always lead me to question installations.

Reply:
Sure, if we agree to keep this option, I'll update the regex string.

Reply:
I like this option since digging through CMakeCache.txt can be annoying. We could also update btest to use this instead as well.